### PR TITLE
Removed hyperlink when availibity is 0 for a clinic.

### PR DESCRIPTION
**Pull request by msandhuNHS**
## Description

Removed hyperlink to prevent navigation to a clinic where availability is 0.

## Type of changes

- [ ] Refactoring (non-breaking change)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)

## Checklist

- [ ] I am familiar with the [contributing guidelines](../docs/CONTRIBUTING.md)
- [ ] I have followed the code style of the project
- [ ] I have added tests to cover my changes
- [ ] I have updated the documentation accordingly
- [ ] This PR is a result of pair or mob programming

---

## Sensitive Information Declaration

To ensure the utmost confidentiality and protect your and others privacy, we kindly ask you to NOT including [PII (Personal Identifiable Information) / PID (Personal Identifiable Data)](https://digital.nhs.uk/data-and-information/keeping-data-safe-and-benefitting-the-public) or any other sensitive data in this PR (Pull Request) and the codebase changes. We will remove any PR that do contain any sensitive information. We really appreciate your cooperation in this matter.

- [ ] I confirm that neither PII/PID nor sensitive data are included in this PR and the codebase changes.

### CODE REVIEW

**Review by Abdul-NHS — APPROVED** [review]

File: src/app/views/clinic_summary/ClinicSummaryTable.jsx [hunk 1]
```diff
@@ -68,19 +68,29 @@ export default function ClinicSummaryTable(props) {
...
                 <tr role="row" class="nhsuk-table__row">
                   <td role="cell" class="nhsuk-table__cell">
-                    <a
-                      id={e.ClinicName.S}
-                      class="nhsuk-back-link__link"
-                      onClick={(event) => onClickClinicHandler(event, e)}
-                    >
-                      {e.ClinicName?.S ? e.ClinicName?.S : "Not Available"}
-                    </a>
+                    {e.Availability.N !== "0" ? (
```
Comment (zainm97) [inline src/app/views/clinic_summary/ClinicSummaryTable.jsx:71]: please address Chengs comment

Comment (msandhuNHS) [reply src/app/views/clinic_summary/ClinicSummaryTable.jsx:71]: It is detected as a String at this point. 0 would not perform the equals operation so a '0' to trigger this condition. 👍

**Review by zainm97 — COMMENTED** [review]
Fab

**Review by zainm97 — APPROVED** [review]
Fab

**Pull request merged**